homeassistant/components/freebox/camera.py

```diff
@@ -0,0 +1,217 @@
+            CONF_INPUT: node["props"]["Stream"],
+            CONF_EXTRA_ARGUMENTS: DEFAULT_ARGUMENTS,
+        }
+        FFmpegCamera.__init__(self, hass, device_info)
+
+        self._supported_features = (
+            CameraEntityFeature.ON_OFF | CameraEntityFeature.STREAM
+        )
+
+        self._command_motion_detection = self.get_command_id(
+            node["type"]["endpoints"], ATTR_DETECTION
+        )
+        self._high_quality_video = self.get_command_id(
+            node["show_endpoints"], ATTR_QUALITY
+        )
+        self._command_flip = self.get_command_id(node["show_endpoints"], ATTR_FLIP)
+        self._motion_threshold = self.get_command_id(
+            node["show_endpoints"], ATTR_THRESHOLD
+        )
+        self._motion_sensitivity = self.get_command_id(
+            node["show_endpoints"], ATTR_SENSITIVITY
+        )
+        self._activation_with_alarm = self.get_command_id(
+            node["show_endpoints"], ATTR_ACTIVATION
+        )
+        self._timestamp = self.get_command_id(node["show_endpoints"], ATTR_TIMESTAMP)
+        self._volume_micro = self.get_command_id(node["show_endpoints"], ATTR_VOLUME)
+        self._sound_detection = self.get_command_id(
+            node["show_endpoints"], ATTR_SOUND_DETECTION
+        )
+        self._sound_trigger = self.get_command_id(
+            node["show_endpoints"], ATTR_SOUND_TRIGGER
+        )
+        self._rstp = self.get_command_id(node["show_endpoints"], ATTR_RTSP)
+        self._disk = self.get_command_id(node["show_endpoints"], ATTR_DISK)
+
+        self.update_node(node)
+
+    @property
+    def supported_features(self) -> CameraEntityFeature:
+        """Flag supported features."""
+        return self._supported_features
+
+    @property
+    def flip(self) -> bool:
+        """Return flip."""
+        return self._flip
+
+    @property
+    def command_flip(self) -> Any | str:
+        """Return command_flip."""
+        return self._command_flip
+
+    def set_flip(self, state: bool) -> None:
+        """Update flip."""
+        self._flip = state
+
+    async def async_flip(self, entity: FreeboxCamera) -> None:
+        """Flip the camera stream."""
+        entity.set_flip(not entity.flip)
+        await entity.set_home_endpoint_value(entity.command_flip, entity.flip)
+
+    @property
+    def motion_detection_enabled(self) -> bool:
+        """Return the camera motion detection status."""
+        return self._attr_motion_detection_enabled
+
+    async def async_enable_motion_detection(self) -> None:
+        """Enable motion detection in the camera."""
+        await self.set_home_endpoint_value(self._command_motion_detection, True)
+        self._attr_motion_detection_enabled = True
+
+    async def async_disable_motion_detection(self) -> None:
+        """Disable motion detection in camera."""
+        await self.set_home_endpoint_value(self._command_motion_detection, False)
+        self._attr_motion_detection_enabled = False
```

Comment:
Should we presume the `_attr_motion_detection_enabled` state as it is now or get the actual value from the API (immediately or at next update) ?

Reply:
Same question for `async_flip`

Reply:
Let's presume it for now